test/json/README.md

```diff
@@ -0,0 +1,28 @@
+The files in this directory were originally copied from the [messageformat project](https://github.com/messageformat/messageformat/tree/11c95dab2b25db8454e49ff4daadb817e1d5b770/packages/mf2-messageformat/src/__fixtures)
+and are here relicensed by their original author (Eemeli Aro) under the Unicode License.
```

Comment:
I'm not sure if this is kosher or not. I will check with legal about the best way to handle this.

Reply:
Based on our discussion with Anne (legal), as sole author you can commit this file, relying on the CLA to cover it.